tests/api/test_category.py

```diff
@@ -235,3 +362,48 @@ def test_category_level(user_api_client, category):
     category_data = content['data']['categories']['edges'][0]['node']
     assert category_data['name'] == child.name
     assert category_data['parent']['name'] == category.name
+
+
+def test_category_image_query(user_api_client, non_default_category):
+    category = non_default_category
+    image_file, image_name = create_image()
+    category.background_image = image_file
+    category.save()
+    category_id = graphene.Node.to_global_id('Category', category.pk)
+    query = """
+        query fetchCategory($id: ID!){
```

Comment:
Could we create some sort of the const for queries that are used in multiple tests?